Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN go mod download
 COPY . .
 
 RUN mkdir -p $GOPATH/pkg/linux_amd64/github.com/iotexproject/ && \
-    make clean build-all
```

Comment:
revert this
`all` includes `test`, which is not necessary, and may fail in docker environment